src/js/api/order.js

```diff
@@ -402,6 +402,114 @@ ripe.Ripe.prototype.setOrderStatus = function(number, status, options, callback)
     return this._cacheURL(options.url, options, callback);
 };
 
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype.setPlayerId = function(playerId, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = this.url + "notify_info" + "?" + this._buildQuery({ player_id: playerId });
+    options = Object.assign(options, {
+        url: url,
+        auth: true,
+        method: "PUT"
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype.setPlayerIdP = function(playerId, options) {
+    return new Promise((resolve, reject) => {
+        this.setPlayerId(playerId, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request));
+        });
+    });
+};
+
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype.getOrderSubscriptionStatus = function(number, options, callback) {

```

Comment:
`isSubscribed` (keep the concepts, if you name it something in the API, call it the same thing here)